### PR TITLE
Change module name to dreamsheets

### PR DESCRIPTION
**Pull request by LucianBuzzo**
This change is a part of productising this repo, and includes an update
to the README.

Change-type: major
Signed-off-by: Lucian Buzzo <lucian.buzzo@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +57,12 @@ $ npm install
 $ npm link @balena/gsheet-utils
```
Comment (AlidaOdendaal) [inline README.md:57]: Change to
```
npm link @balena/dreamsheets
```
or
```
npm link dreamsheets
```
depending on the value of `name` in `package.json`.

File: package.json [hunk 2]
```diff
@@ -1,22 +1,24 @@
 {
-  "name": "@balena/gsheet-utils",
+  "name": "dreamsheets",
```
Comment (AlidaOdendaal) [inline package.json:2]: Should this be

`"name": "@balena/dreamsheets",`

?

**Review by AlidaOdendaal — DISMISSED** [review]
Looks great! Just a couple of small comments.

**Review by pranasziaukas — COMMENTED** [review]
I don’t mind the playful name or logo, so LGTM in that regard.

I do have the same questions as Alida pointed out in her review.

File: package.json (hunk 2 above)
Comment (LucianBuzzo) [reply package.json:2]: @AlidaOdendaal I don't think so, scoping the package in this way is useful for private packages, and for branding purposes if the product is directly linked to the balena platform. In this case I don't think it's necessary.

File: README.md (hunk 1 above)
Comment (LucianBuzzo) [reply README.md]: @AlidaOdendaal I've updated to `npm link dreamsheets`

**Review by somombo — APPROVED** [review]
ltgm

File: README.md (hunk 1 above)
Comment (somombo) [reply README.md]: @AlidaOdendaal's review is resolved, so I'll save her a trip here and self-certify.

**Comment by somombo** [comment]
@balena-ci

**Comment by somombo** [comment]
@balena-ci I self certify!

**Comment by somombo** [comment]
@balena-ci I self certify!

**Comment by LucianBuzzo** [comment]
@somombo This catches me out all the time, self-certify fails with an error if *you're not the original author*. You can approve in the top right though

**Comment by LucianBuzzo** [comment]
@balena-ci I self-certify!

**Pull request merged**